code/__DEFINES/logging.dm

```diff
@@ -56,22 +56,97 @@
 
 // Log header keys
 #define LOG_HEADER_CATEGORY "category"
+#define LOG_HEADER_CATEGORY_LIST "category-list"
 #define LOG_HEADER_INIT_TIMESTAMP "timestamp"
-#define LOG_HEADER_ROUND_ID "round_id"
+#define LOG_HEADER_ROUND_ID "round-id"
+#define LOG_HEADER_SECRET "secret"
 
 // Log data keys
+#define LOG_ENTRY_CATEGORY "category"
 #define LOG_ENTRY_MESSAGE "message"
 #define LOG_ENTRY_TIMESTAMP "timestamp"
 #define LOG_ENTRY_DATA "data"
+#define LOG_ENTRY_SCHEMA_VERSION "schema-version"
+#define LOG_ENTRY_WORLD_DATA "world-state"
 
 // Log json keys
 #define LOG_JSON_CATEGORY "category"
 #define LOG_JSON_ENTRIES "entries"
-#define LOG_JSON_LOGGING_START "log_start"
+#define LOG_JSON_LOGGING_START "log-start"
 
-// Log categories
-#define LOG_CATEGORY_NOT_FOUND "invalid_category"
-#define LOG_CATEGORY_TARGET_ZONE_SWITCH "target_zone_switch"
+// Category for invalid/missing categories
+#define LOG_CATEGORY_NOT_FOUND "invalid-category"
+
+// Misc categories
+#define LOG_CATEGORY_ATTACK "attack"
+#define LOG_CATEGORY_CONFIG "config"
+#define LOG_CATEGORY_DYNAMIC "dynamic"
+#define LOG_CATEGORY_ECONOMY "economy"
+#define LOG_CATEGORY_FILTER "filter"
+#define LOG_CATEGORY_MANIFEST "manifest"
+#define LOG_CATEGORY_MECHA "mecha"
+#define LOG_CATEGORY_PAPER "paper"
+#define LOG_CATEGORY_SHUTTLE "shuttle"
+#define LOG_CATEGORY_SILICON "silicon"
+#define LOG_CATEGORY_SIGNAL "signal"
+#define LOG_CATEGORY_SPEECH_INDICATOR "speech-indiciator"
+#define LOG_CATEGORY_SUSPICIOUS_LOGIN "supicious-login"
+#define LOG_CATEGORY_TARGET_ZONE_SWITCH "target-zone-switch"
+#define LOG_CATEGORY_TELECOMMS "telecomms"
+#define LOG_CATEGORY_TOOL "tool"
+#define LOG_CATEGORY_VIRUS "virus"
+
+// Admin categories
+#define LOG_CATEGORY_ADMIN "admin"
+#define LOG_CATEGORY_ADMIN_CIRCUIT "admin-circuit"
+#define LOG_CATEGORY_ADMIN_DSAY "admin-dsay"
+
+// Admin private categories
+#define LOG_CATEGORY_ADMIN_PRIVATE "admin-private"
+#define LOG_CATEGORY_ADMIN_PRIVATE_ASAY "admin-asay"
```

Comment:
Could you please Rename admin-private to adminprivate so public logs can easy parse `-` as a analog for `LOGTYPE: LOGSUBTYPE:` and i can reuse the same switch tree for both file types when i add support to public logs.

asay should also be renamed to `"adminprivate-asay"` to keep it inline with the above as well.

basically

Reply:
gotcha